osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/main/java/org/opensmartgridplatform/adapter/protocol/iec60870/domain/entities/Iec60870Device.java

```diff
@@ -27,18 +31,33 @@
     @Column(unique = true, nullable = false, length = 40)
     private String deviceIdentification;
 
+    @Column(nullable = false)
+    @Enumerated(EnumType.STRING)
+    private DeviceType deviceType;
+
+    @Column
+    private String gatewayDeviceIdentification;
```

Comment:
Gatewaydevice is placed here as well in order to prevent access to osgp-core database, while still being able to send light measurement device specific events to osgp-core (reusing existing osgp-core event functionality)